learn2learn/utils/lightning.py

```diff
@@ -32,38 +154,22 @@ def __init__(
         self.test_tasks = test_tasks
         self.epoch_length = epoch_length
 
-    @staticmethod
-    def epochify(taskset, epoch_length):
-        class Epochifier(object):
-            def __init__(self, tasks, length):
-                self.tasks = tasks
-                self.length = length
-
-            def __getitem__(self, *args, **kwargs):
-                return self.tasks.sample()
-
-            def __len__(self):
-                return self.length
-
-        return Epochifier(taskset, epoch_length)
-
     def train_dataloader(self):
-        return EpisodicBatcher.epochify(
+        return Epochifier(
```

Comment:
Was `Epochifier` inadvertently removed? Or should it be replaced by `TaskDataParallel`?

Also, if I remember correctly `Epochifier` was preventing us from using pytorch_lightning > 1.0.2. I believe the fix was just to inherit from a data class in lightning. @nightlessbaron do you remember which class and if that was enough?

Reply:
@seba-1511, quick answer to your first question: `Epochifier` is replaced by `TaskDataParallel`

Reply:
@seba-1511, as @pietrolesci mentioned, in the benchmark, we replaced `Epochifier` with `TaskDistributedParallel`. The downside is windows users can't make use of it as stated in [#5358](https://github.com/PyTorchLightning/pytorch-lightning/issues/5358). 

> I believe the fix was just to inherit from a data class in lightning.
I couldn't figure it out. That is still unresolved.

Reply:
Hey @seba-1511, yes it was inadvertently removed.

However, I added a TODO. It should be provided replaced by the TaskDataParallel, TaskDistributedDataParallel when tested on your side.

I will copy those files on the Flash side to no block the integration until the next learn2learn release.

Best,
T.C